### ISSUE TITLE
Gated Marketplaces: Owner approval

### ISSUE THREAD

**Issue by didiermis** [hashed-io/hashed-substrate#121]
Before users can list a tax credit for sale, they must FIRST receive approval from the marketplace owner:
- [ ] Waiting on #122
- [ ] Create a new storage map to track owner approval
- [ ] Create an extrinsic in the gated-marketplace pallet to allow the owner to unfreeze the tax credit
- [ ] Once the owner has approved the tax credit, the item can be put up for sale

**Comment by didiermis** [comment]
No longer needed.
